diff --git a/proton b/proton
index c74bd571..2907b948 100755
--- a/proton
+++ b/proton
@@ -1348,6 +1348,7 @@ def default_compat_config():
                 "2176900", #Fablecraft
                 "2853730", #Skull and Bones
                 "1761380", #Otherworld Legends (just to disable nvcuda.dll)
+                "2079120", #Warudo
                 ]:
             ret.add("disablenvapi")
 
@@ -1378,6 +1379,7 @@ def default_compat_config():
 
         if appid in [
                 "2698940", #The Crew Motorfest
+                "2079120", #Warudo
                 ]:
             ret.add("hidenvgpu")
 
